nixtla/nixtla_client.py

```diff
@@ -620,6 +621,11 @@ def ensure_contiguous_arrays(d: Dict[str, Any]) -> None:
 
         ensure_contiguous_arrays(payload)
         content = orjson.dumps(payload, option=orjson.OPT_SERIALIZE_NUMPY)
+        content_size_mb = sys.getsizeof(content) / (1024 * 1024)
```

Comment:
`content` is already a bytes object, we just want to count the number of bytes, not get the size of the python object.
```suggestion
        content_size_mb = len(content) / (1024 * 1024)
```